### PR TITLE
A way to disable floating_ips

### PR DESCRIPTION
**Pull request by martbhell**
Bonus changes:
 - A way to define more settings without having to modify the template
   in this role every time (would it be nicer to use ini_file in some way?)

### CODE REVIEW

File: templates/tempest.j2 [hunk 1]
```diff
@@ -24,6 +24,16 @@ volume_size = 10
...
+{% if rally_tempest_floating_ips is defined %}
+floating_ips = {{ rally_tempest_floating_ips }}
+{% endif %}
 
 [identity]
 ca_certificates_file = {{ tempest_ca_path }}
+
+{% if rally_tempest_extra_settings is defined %}
+# Extra tempest.conf settings
+{% for param in rally_tempest_extra_settings %}
```
Comment (oscarkraemer) [inline templates/tempest.j2]: Is there a guarantee that all the settings will be added in the same order as they are defined?

Comment (oscarkraemer) [reply templates/tempest.j2]: Also this feels a bit hackerish.

Comment (martbhell) [reply templates/tempest.j2]: Yes. I don't know what that is. The data I think is in YAML lists which are different than python lists?

**Comment by martbhell** [comment]
I think you are wrong and I have done it like this in other roles[1] and they end up in the same order.
I agree that it's hackerish and I'm proud of it :) 

One could perhaps use ini_file to manage the whole file like that. But I have a feeling it would be a lot slower than using the template module.

[1]: https://github.com/CSCfi/ansible-role-slurm/blob/master/templates/slurm.conf.j2#L144-L149

**Comment by oscarkraemer** [comment]
Yeah, it seems that I'm wrong. Thumb of rule: lists are always ordered, dictionary/hashes never in ordered (for humans). 

Okay, :) you can continue to feel proud.

**Pull request merged**